contracts/pcv/curve/StableSwapOperatorV1.sol

```diff
@@ -75,9 +75,8 @@ contract StableSwapOperatorV1 is PCVDeposit {
         uint256 _minimumRatioThreshold,
         uint256 _maximumRatioThreshold
     ) CoreRef(_core) {
-        require(minimumRatioThreshold < maximumRatioThreshold, "Min ratio must be less than max ratio");
-        require(minimumRatioThreshold >= 0.01e18, "Min ratio must be at least 1%.");
-        require(maximumRatioThreshold <= 100e18, "Max ratio cannot be above 100x.");
+        setMinRatio(_minimumRatioThreshold);
```

Comment:
Should call the internal ones, caller is unlikely to be a gov

Reply:
Fixed